tasks/create.yaml

```diff
@@ -63,4 +63,4 @@ tasks:
           CUSTOM_PEPR_IMAGE=$( [ "${FLAVOR}" = "registry1" ] && echo "--custom-image ${REGISTRY1_PEPR_IMAGE}" ) || CUSTOM_PEPR_IMAGE=""
           rm -fr dist
           npm ci
-          npx pepr build $CUSTOM_PEPR_IMAGE
+          npx pepr build $CUSTOM_PEPR_IMAGE -z chart
```

Comment:
It seems like the chart does not pull our `env` from the `package.json` so we will either need to:
1. Revert this change, just use the manifest for now.
2. Add override values in our zarf.yaml that specify all the default env + our additional ones. This is viable but I'd be worried about us missing changes to the defaults on upgrades.

I think I'd lean towards 1, and we should open a pepr issue regarding the mismatch in behavior between manifest and chart.

Reply:
In the interest of time i'll remove the change to using the chart. Per our other discussion you'll create an issue in pepr for the chart generation to include the extra env things.

Reply:
FYSA opened https://github.com/defenseunicorns/pepr/issues/951 based on this issue.